### DISCUSSION TITLE
Add Container to PWA

### DISCUSSION THREAD

**Discussion by GunGunGun** [Floorp-Projects/Floorp#1395] [Feature Request]
Currently PWAs created using Install site as webapp is using the main container (Default Container) so it shares the same account with the main container, please add Container to make it works like web app instead.

**Comment by SystemError12** [comment]
Big plus from me. 

This option was availabe a few months ago and it suddenly stopped working. Please bring it back.

**Comment by CutterKnife** [comment]
Related issue: https://github.com/Floorp-Projects/Floorp/issues/1085

**Comment by MurzNN** [comment]
Oh, the related issue is closed, but still can't find a way to assign a WebApp (PWA) to a container on the Floorp Hub Web Apps configuration page.

If I'm doing this using the "Always open this site in..." feature of the Multi-Account containers, Floorp just starts to open an empty window, and the real page is opened in a new tab instead of this window.

**Comment by Ryosuke-Asano** [comment]
https://github.com/Floorp-Projects/Floorp/pull/2443
